lib/Doctrine/Record.php

```diff
@@ -2382,15 +2382,15 @@ public function loadReference($name)
     /**
      * call
      *
-     * @param string|array $callback    valid callback
+     * @param callable $callback    valid callback
      * @param string $column            column name
-     * @param mixed arg1 ... argN       optional callback arguments
-     * @return Doctrine_Record provides a fluent interface
+     * @param mixed ...$args       optional callback arguments
+     * @return $this provides a fluent interface
      */
-    public function call($callback, $column)
+    public function call($callback, $column, ...$args)
```

Comment:
Changing this one to use variadic for params 3+.

I did confirm all of the logic in here (`func_get_args` returns all args, even the named ones, and the previous code removes the first with `array_shift` (the `$callback` param) so `$args` ends up being params 2+).

This is also covered by tests: https://codecov.io/gh/diablomedia/doctrine1/src/master/lib/Doctrine/Record.php#L2390